faq.rst

```diff
@@ -161,3 +161,40 @@ onto the users stack.
 A simple example of this in practice is provided in the
 ``sample_part1_callback.ipynb`` in `pyEXP-examples repo
 <https://github.com/EXP-code/pyEXP-examples>`_.
+
+What do these parameters mean?
+--------------------------------------
+
+There are a lot of parameters that you can set in EXP/pyEXP! We describe
+some of them below.
+
+``numr`` is the number of radial grid points for spherical expansion, this
+sets the interpolation length between grid points. There is really no downside to making this 
+large, but anything larger than ~2000 is overkill. This generally doesn't need to
```

Comment:
Actually, there is a downside.   If `numr` is too large, the interpolation can suffer from truncation error.  Otherwise I agree about overkill.  I'd suggest sticking to 10000, tops to be safe from truncation.